lib/models/addon.js

```diff
@@ -851,12 +851,7 @@ var Addon = CoreObject.extend({
   addonJsFiles: function(tree) {
     this._requireBuildPackages();
 
-    var includePatterns = this.registry.extensionsForType('js').map(function(extension) {
-      return new RegExp(extension + '$');
-    });
-
     return new Funnel(tree, {
-      include: includePatterns,
```

Comment:
I didn't see anything either, other than the fact that it would be present. I am interested in letting this ride.